src/traces/table/plot.js

```diff
@@ -0,0 +1,892 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var c = require('./constants');
+var d3 = require('d3');
+var gup = require('../../lib/gup');
+var Drawing = require('../../components/drawing');
+var extendFlat = require('../../lib/extend').extendFlat;
+var svgUtil = require('../../lib/svg_text_utils');
+var raiseToTop = require('../../lib').raiseToTop;
+var cancelEeaseColumn = require('../../lib').cancelTransition;
+
+module.exports = function plot(gd, calcdata) {
+
+    if(c.clipView) {
+        gd._fullLayout._paper.attr('height', 2000);
```

Comment:
`clipView: false` in constants.js. So is this block only useful for debugging? It so, :hocho: please.

Reply:
Yes, invaluable for development even to the last touches, planning to 🔪 it just before the merge